torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -665,6 +667,35 @@ void AliasDb::analyzeWait(Node* node) {
   }
 }
 
+void AliasDb::analyzeBackward(Node* node) {
```

Comment:
This function doesn't handle a ton of aliasing cases. For instance, what if the grad tensor is stored in a list inside a class somewhere. A better way to do this would be to mark backward as CONSERVATIVE.